packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -261,11 +265,20 @@ export async function pollDeploymentStatus(deployCtx: DeployContext) {
       polledOperations = [];
     } catch (error) {
       tryCount++;
-      if (tryCount > failedCount) {
-        throw error;
+      if (tryCount < MaxLogTimes) {
+        deployCtx.ctx.logProvider?.warning(
+          `[${PluginDisplayName.Solution}] ${deployCtx.deploymentName} -> waiting to get deplomyment status [Retry time: ${tryCount}]`
+        );
       }
-      deployCtx.logProvider?.warning(
-        `[${PluginDisplayName.Solution}] ${deployCtx.deploymentName} -> waiting to get deplomyment status [${tryCount}]`
+      const pollError = returnSystemError(
+        error,
+        SolutionSource,
+        SolutionError.FailedToPollArmDeploymentStatus
+      );
+      sendErrorTelemetryThenReturnError(
```

Comment:
we may just send the error telemetry when exceeds max retry count to reduce possible noises.

Reply:
It is fixed. Thanks.